tools/wpt/browser.py

```diff
@@ -290,7 +290,7 @@ def find_webdriver(self, channel=None):
 
     def get_version_and_channel(self, binary):
         version_string = call(binary, "--version").strip()
-        m = re.match(r"Mozilla Firefox (\d+\.\d+(?:\.\d+)?)(a|b)?", version_string)
+        m = re.match(br"Mozilla Firefox (\d+\.\d+(?:\.\d+)?)(a|b)?", version_string)
         if not m:
             return None, "nightly"
         version, status = m.groups()
```

Comment:
I don't think this works. You're using a bytes regex, so the captured groups are also bytes, but the keys in `channel` are strings, so `channel.get(status)` would never succeed.

Reply:
Ahh... this is not gonna to work. I might need to do decode.

Reply:
This is now addressed in https://github.com/web-platform-tests/wpt/pull/21656